server/storage.go

```diff
@@ -316,6 +315,25 @@ func (s *S3Storage) Put(token string, filename string, reader io.Reader, content
 	return
 }
 
+func (s *S3Storage) Put(token string, filename string, reader io.Reader, contentType string, contentLength uint64) (err error) {
+	key := fmt.Sprintf("%s/%s", token, filename)
```

Comment:
you have now `key` defined two times: here and at https://github.com/dutchcoders/transfer.sh/pull/201/files#diff-8cf047e4c4eb7b64ea58583ff04a3110R205

please, remove the second and refactor `PutMulti` method to accept directly a key param

Reply:
Refactored 👍